runtime/jvm/src/main/kotlin/kotlinx/serialization/java/DateSerializer.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.java
+
+import kotlinx.serialization.*
+import kotlinx.serialization.internal.*
+import java.text.*
+import java.util.*
+
+/**
+ * Serializer for [java.util.Date] which serializers date as format string in default US locale.
+ */
+public object DateSerializer : KSerializer<Date> {
+    private val format = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, Locale.US)!!
```

Comment:
Why a non standard format, without a time zone reference?

Reply:
And, there are actually a number of different relevant standards. You probably want to create serializers for each of them. As well as with or without time. Of course maybe dates/calendars/instants (whichever date type you want to use) are not the best starting point for this.